src/applications/SimulationSpecification.java

```diff
@@ -35,13 +36,40 @@ public int getChangeOverTimes(int machineNumber) {
     public void setSpecificationsForTasks(int jobNumber, int[] specificationsForTasks) {
         jobSpecifications[jobNumber].setSpecificationsForTasks(specificationsForTasks);
     }
+//
+//    public void setJobSpecification(JobSpecification[] jobSpecifications) {
+//        this.jobSpecifications = jobSpecifications;
+//    }
+//
+//    public void setJobSpecification(int tasksNumber, int[] tasks) {
+//        this.jobSpecifications = jobSpecifications;
+//    }
+//
+//    public JobSpecification getJobSpecifications(int jobNumber) {
+//        return jobSpecifications[jobNumber];
+//    }
```

Comment:
Is this necessary to have still?

Reply:
I agree with Chuck. Maybe this should be removed?